uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-widget/leave-widget-next-leave.component.spec.js

```diff
@@ -133,11 +135,6 @@ define([
           expect(ctrl.nextLeaveRequest).toEqual(expectedNextLeave);
         });
 
-        it('converts the request dates from strings to Date objects', function () {
-          expect(moment.isMoment(ctrl.nextLeaveRequest.from_date)).toBe(true);
```

Comment:
This test was testing the type, why it was deleted?

Reply:
I added these two lines to fix the failing test case
```
expectedNextLeave.from_date = moment(expectedNextLeave.from_date);
expectedNextLeave.to_date = moment(expectedNextLeave.to_date);
```

So https://github.com/civicrm/civihr/pull/2296/files#diff-58556ef72eedbf30b19477254d60fa52R135 line takes care of the deleted test case.